sp800-63d/vot_mapping.md

```diff
@@ -15,21 +15,19 @@ The trustmark URI for these definitions is [[ URI for this document ]].
 
 This document uses the four components defined in VoT: Identity Proofing (P), Primary Credential Usage (C), Primary Credential Management (M), and Assertion Presentation (A).
 
-The VoT components P, C, and A correspond to 800-63's Identity Assurance Level (IAL), Authenticator Assurance Level (AAL), and Federation Assurance Level (FAL), respectively. While M does not correspond directly to an 800-63 assurance level category, this document defines a set of M values derived from requirements and recommendations in 800-63. 
+The VoT components P, C, and A correspond to 800-63's Identity Assurance Level (IAL), Authenticator Assurance Level (AAL), and Federation Assurance Level (FAL), respectively. While M does not correspond directly to an 800-63 assurance level category, this document defines a set of M values derived from requirements and recommendations in 800-63. This document does not define any additional component categories.
 
-This document uses terms as defined in 800-63-3. Some definitions differ from those in VoT. Notably, the term "authenticator" herein has the same meaning as "credential" in VoT.
-
-This document does not define any additional component categories. Any additional categories SHALL NOT be used.
+This document uses terms as defined in 800-63-3, Appendix A - Definitions and Abbreviations. Some definitions differ from those in VoT. Notably, the term "authenticator" herein has the same meaning as "credential" in VoT.
 
 ## Component Values
 
-This document does not use any of the existing values in VoT but instead defines new component values in accordance with Section 8 of VoT.
+This document defines new component values in accordance with Section 8 of VoT.
 
 ### Identity Proofing
 
-#### IAL to VoT
+IAL maps into the VoT component P using a numeric identifier. Identity proofing characteristics map into P using an alphabetic identifier.
```

Comment:
I think these intro paragraphs were better where they were before, because we need to be more specific. Only one numeric, possibly multiple alphabetic, etc. Even though it's all covered below I'd be afraid of that important aspect being missed.